test/gen-cert

```diff
@@ -63,7 +63,12 @@ cat >> req.conf <<eof
 [ v3_req ]
 basicConstraints = CA:FALSE
 keyUsage = nonRepudiation, digitalSignature, keyEncipherment, keyCertSign
-subjectAltName = DNS.1:${DOMAIN}
+#subjectAltName = DNS.1:${DOMAIN}
```

Comment:
Since the DOMAIN passed from `test/setup` script was `openstack.example.com`, I was trying to use multiple entries for the Subject Alternative Names to enable swift fqdn as follows:
```
subjectAltName = @alt_names
[alt_names]
DNS.1 = ${DOMAIN}
DNS.2 = swift.${DOMAIN}
```

But I thought it again and want to use a better approach:
1. Pass `*.openstack.example.com` as DOMAIN to `gen-cert`.
2. Remove "*." from above DOMAIN as a FILE_PREFIX, and use it instead of original DOMAIN for the name of temp files for cert generating.